File: Software Development Module/Revision/Players/DefensivePlayer.java
/*
DefensivePlayer.java
Billy Cussen
28/07/2020
*/

public class DefensivePlayer extends FootballPlayer{

    //Constructor
    public DefensivePlayer(String name){
        super(name);
    }

    @Override
    public String role(){
        return "Hi, I'm "+name+". I play in Defense.";
    }

}

File: Python Course/Software Development Module - in Python/Revision/Players/DefensivePlayer.py
"""
DefensivePlayer.py
Billy Cussen
26/10/2020
"""
import FootballPlayer

class myDefensivePlayer(FootballPlayer.myFootballPlayer):
    def __init__(self, name):
        super().__init__(name)

    def getRole(self):
        return "Hi "+str(self.name)+", I'm a Defensive Player"